src/tests/class_tests/openms/source/HyperScore_test.cpp

```diff
@@ -75,22 +75,24 @@ START_SECTION((static double compute(double fragment_mass_tolerance, bool fragme
   {
     p.setMZ(i);
     rp.setMZ(i);
+    rp.setMetaValue("IonName", String("y") + String(i));
     exp_spectrum.push_back(p);
     theo_spectrum.push_back(rp);
   }
-  TEST_REAL_SIMILAR(HyperScore::compute(0.1, false, exp_spectrum, theo_spectrum), 3.609438);
-  TEST_REAL_SIMILAR(HyperScore::compute(10, true, exp_spectrum, theo_spectrum), 3.609438);
+  TEST_REAL_SIMILAR(HyperScore::compute(0.1, false, exp_spectrum, theo_spectrum), 7.39693);
```

Comment:
most likely because the ion annotation was missing in the test and it got interpreted as b-ions